crs_motion_planning/src/path_planning_utils.cpp

```diff
@@ -363,6 +562,7 @@ bool crsMotionPlanner::generateSurfacePlans(pathPlanningResults::Ptr& results)
                                     post_speed_split_trajs[i].points.end());
       if (!findClosestJointOrientation(end_orig, new_raster_end, end_new, config_->descartes_config.axial_step))
       {
+        ret_app_addable = false;
```

Comment:
This variable name makes it hard to tell how it's used

Reply:
it's supposed to mean both the retreat (ret) and approach (app) are reachable (addable). If either is not reachable then it tosses out that segment.